src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -825,16 +813,11 @@ namespace WaterToAirHeatPumpSimple {
 
 		if ( ( SensLoad != 0.0 || LatentLoad != 0.0 ) && ( Node( AirInletNode ).MassFlowRate > 0.0 ) ) {
 
-			// changed the water mass flow rate to be equal to the design times run time fraction in order to account for
-			// cycling of equipment
-			SimpleWatertoAirHP( HPNum ).WaterMassFlowRate = SimpleWatertoAirHP( HPNum ).DesignWaterMassFlowRate * WaterPartLoad;
-
-			//    SimpleWatertoAirHP(HPNum)%WaterMassFlowRate =    SimpleWatertoAirHP(HPNum)%DesignWaterMassFlowRate
-
 			// Model requires the values to be calculated at full design flow rate for air and then scaled to part load ratio.
 			// So always start the calculations by setting the air flow rate to design flow rate.
 
-			//    SimpleWatertoAirHP(HPNum)%AirMassFlowRate   = Node(AirInletNode)%MassFlowRate
+			SimpleWatertoAirHP( HPNum ).WaterMassFlowRate = SimpleWatertoAirHP( HPNum ).DesignWaterMassFlowRate;
+
```

Comment:
Underlying cause of issue. Water flow rate was calculated in proportion to RTF in Init (only for cycling water flow mode case). This caused 2 problems, 1) the full load capacity was calculated using the lowered water flow (in Calc), and 2) the outlet water temp would be excessive at low RTF.